system-int-test/test/cmr/system_int_test/ingest/collection_ingest_test.clj

```diff
@@ -77,7 +77,7 @@
           {:keys [concept-id revision-id]} (ingest/ingest-concept concept {:token launchpad-token})]
       (index/wait-until-indexed)
       (is (mdb/concept-exists-in-mdb? concept-id revision-id))
-      (is (= 1 revision-id)))))
+      (is (= 6 revision-id)))))
```

Comment:
This test wasn't running because it had the same name as the namespace or?

Reply:
Yea I think so.